examples/data-sources/b1ddi_dns_auth_nsgs/data-source.tf

```diff
@@ -16,3 +16,10 @@ data "b1ddi_dns_auth_nsgs" "tf_example_auth_nsg" {
 
 # Get all DNS Auth NSGs
 data "b1ddi_dns_auth_nsgs" "all_dns_auth_nsgs" {}
+
+# Get DNS Auth NSG with the specified tags
```

Comment:
Plural should be used for consistency, e.g. `DNS Auth NSGs` instead of `DNS Auth NSG`.

Reply:
Fixed

Reply:
My comment was about the comment on line 19 which hasn't been modified.

Reply:
Still not addressed.

Reply:
line 19 has been modified in the latest commit .

Reply:
this was addressed in the latest commit !

Reply:
My bad, I might have looked at a previous commit.